users/config.rst

```diff
@@ -331,6 +331,9 @@ order
     the 1 KB becomes known to the pulling device.
 
 ignoreDelete
+    .. warning::
+        Enabling this is highly not recommended - use at your own risk.
```

Comment:
Enabling this will willingly corrupt the database and lead to permanently out of sync state - use at your own risk